rtf

```diff
@@ -413,177 +415,50 @@ class Vagrant:
-          pf_id = int(m.group(2))
-          while not re.match(r'### END OF AUTOGENERATION FOR ' + name, line):
-            line = fd.readline()
-          lines.extend(host_lines(name, pf_id))
-          lines.append(line) # re-add the footer
-          updated = True
-        # no existing platform, create a new one
-        elif not updated and re.match(r'### AUTOGEN TAG', line):
-          # Look for next id, default case would be to get max+1
-          next_id = max_pf_id + 1
-          # But we prefer to use min - 1
-          if min_pf_id > next_id:
-            min_pf_id = next_id
-          if min_pf_id > 1:
-            next_id = min_pf_id - 1
-          else:
-            # If min = 1, tries to find and id between two used ids
-            for x in range(1, max_pf_id):
-              if not x in ids:
-                next_id = x
-                break
-          if next_id >= 256:
-            print(" New platform id will be "+next_id+" which will fail (invalid ip, ...) , please delete some platform before creating a new one")
-            exit(6)
-          lines.append('### AUTOGENERATED FOR PLATFORM ' + name + " (" + str(next_id) + ") https://localhost:" + str(80+next_id) + "81/ \n")
-          lines.extend(host_lines(name, next_id))
-          lines.append('### END OF AUTOGENERATION FOR ' + name + "\n")
-          lines.append("\n### AUTOGEN TAG\n")
-          updated = True
-        # unknown line, keep it
-        else:
-          lines.append(line)
-        line = fd.readline()
-      # rewrite the file
-      fd.seek(0)
-      fd.truncate()
-      fd.writelines(lines)
-
-  @staticmethod
-  def remove_platform(name):
-    """ Update or replace the Vagrantfile configuration by removing the given platform """
-    lines = []
-    # parse Vagrantfile into the lines array (and update it)
-    with open("Vagrantfile", "r+") as fd:
-      updated = False
-      line = fd.readline()
-      matched = False
-      while line:
-        if re.match(r'### AUTOGENERATED FOR PLATFORM ' + name, line):
-          matched = True
-        if not matched:
-          lines.append(line)
-        if re.match(r'### END OF AUTOGENERATION FOR ' + name, line):
-          # Remove trailing new line
-          line = fd.readline()
-          matched = False
-        line = fd.readline()
-      # rewrite the file
-      fd.seek(0)
-      fd.truncate()
-      fd.writelines(lines)
-
 class Platform:
   """ A test platform
   Can be setup or teared down at once
   Can be used to run a test scenario
   """
-  def __init__(self, name, platform_info, override):
+  def __init__(self, name, override={}):
     self.name = name
     self.hosts = {}
-    self.plugins = {}
     self.provider = "virtualbox"
+    self.override = override
+    self.has_relay = False
+    platform_info = load_json("platforms/" + name + ".json")
 
-    # Initialized at setup time
-    script_dir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
I think we brake rtf test compatibility in a PR before, this is mandatory to run a scenario since the spec_helper expect rtf to pass the rudder-cli path via ENV

Reply:
I don't see those variables used anywhere in the scenarii